randomgen/generator.pyx

```diff
@@ -572,7 +572,7 @@ cdef class RandomGenerator:
                 randoms_data[i] = random_positive_int(self._brng)
         return randoms
 
-    def randint(self, low, high=None, size=None, dtype=int):
+    def randint(self, low, high=None, size=None, dtype=int, bint use_masked_generator=True):
```

Comment:
No need to type here since this is always object

Reply:
👍

Reply:
Are you happy with the order of the params here? I'll do more PRs to   enable the _rand_int64, _rand_int16, etc. callees to all also make use of the use_masked_generator flag.

Reply:
Yes.  This is backward compat for anyone who uses positions only, which is important.  I think we should either have `use_masked=True` since it is as clear as `use_masked_generator` and saves typing.  Eventually, plan will be to add this, in a version or two switch the default to `False`, and then consider removing the masked option all together and have only lemire.  Is is how NumPy makes changes like this.